instrumentation/opentelemetry-instrumentation-elasticsearch/src/opentelemetry/instrumentation/elasticsearch/__init__.py

```diff
@@ -138,7 +138,7 @@ def _instrument(self, **kwargs):
         response_hook = kwargs.get("response_hook")
         _wrap(
             elasticsearch,
-            "Transport.perform_request",
+            "AsyncTransport.perform_request",
```

Comment:
This is not the right solution.

Reply:
Yes working on the rest of the code. thanks for keeping an eye on this.

Reply:
Please convert this to draft if it's not ready for reviews yet.

Reply:
Do we need to add support for AsyncTransport ?

Reply:
That is the feature request. Did you read the issue? You don't replace one with other. And there should be tests.